spec/commands/get_review_recommendations_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+describe GetReviewRecommendations do
+  def perform(*args)
+    described_class.for(*args)
+  end
+
+  def perform_with_predefined_args
+    perform(
+      github_user_id: github_user_id,
+      other_users_ids: other_users_ids
+    )
+  end
+
+  def times_reviewed_by_each_user(*args)
+    described_class.for(*args)
+  end
+
+  def times_reviewed_by_each_user_with_predefined_args
+    times_reviewed_by_each_user(
+      github_user_id: github_user_id,
+      other_users_ids: other_users_ids
+      )
+  end
+
+  let!(:requesting_user) { create(:github_user) }
+  let!(:other_user1) { create(:github_user) }
+  let!(:other_user2) { create(:github_user) }
+  let!(:other_user3) { create(:github_user) }
+  let!(:other_user4) { create(:github_user) }
+  let!(:other_user5) { create(:github_user) }
+  let!(:other_user6) { create(:github_user) }
+  let!(:other_user7) { create(:github_user) }
+  let!(:github_user_id) { requesting_user.id }
+
+  context 'no other users in the team' do
+    let(:other_users_ids) {[]}
+
+    it 'returns empty list of most recommended users' do
+      expect(perform_with_predefined_args[:best]).to eq([])
+    end
+
+    it 'returns empty list of least recommended users' do
+      expect(perform_with_predefined_args[:worst]).to eq([])
+    end
+  end
+
+  context '3 or less other users in the team' do
+    let(:other_users_ids) { [other_user1.id, other_user2.id, other_user3.id] }
+
+    it 'returns list with all other members as most recommended users' do
+      expect(perform_with_predefined_args[:best]).to eq([other_user1, other_user2, other_user3])
```

Comment:
acá no me acuerdo qué pasa cuando se retornan desordenados. Creo que el test podría fallar. Lo que hago en ese caso es en vez de usar `eq(arreglo)` es usar `expect(...).to contain_exactly(other_user1, other_user2, other_user3)`. Así el test no falla por tonteras